src/webgpu/api/validation/resource_usages/texture/in_render_misc.spec.ts

```diff
@@ -571,3 +572,79 @@ g.test('subresources,texture_usages_in_copy_and_render_pass')
       encoder.finish();
     }, false);
   });
+
+g.test('subresources,texture_view_usages')
+  .desc(
+    `
+  Test that the usages of the texture view are used to validate compatibility in command encoding
+  instead of the usages of the base texture.`
+  )
+  .params(u =>
+    u
+      .combine('usage', ['color-attachment', ...kTextureBindingTypes] as const)
```

Comment:
nit: `usage` isn't a GPUTextureUsage, probably should be named something like `bindingType`